examples/svelte-apollo/src/apollo.js

```diff
@@ -0,0 +1,56 @@
+import ApolloClient from "apollo-client";
+import { NhostClient } from '@nhost/nhost-js';
+import { InMemoryCache } from "apollo-cache-inmemory";
+import { WebSocketLink } from "apollo-link-ws";
+import { split } from "apollo-link";
+import { HttpLink } from "apollo-link-http";
+import { getMainDefinition } from "apollo-utilities";
+require('dotenv').config()
+
+const nhost = new NhostClient({
+  backendUrl: process.env.backend_uri,
+});
+
+const accessToken = nhost.auth.getAccessToken()
+
+const headers = {
+  'content-type': 'application/json',
+  'x-hasura-admin-secret': accessToken
```

Comment:
1. The header format should be: `Authorization: Bearer <accessToken>` [[docs](https://hasura.io/docs/latest/graphql/core/auth/authentication/jwt/#tldr)]
2. The `Authorization` header should only be set if the user is signed in.

Reply:
Sure I've made changes will update PR soon.